src/applications/personalization/profile/tests/e2e/personal-and-contact-info/mailing-address.cypress.spec.js

```diff
@@ -65,21 +75,19 @@ const confirmWebAddressesAreBlocked = () => {
 
   cy.findByRole('textbox', { name: /street address.*required/i })
     .clear()
-    .type('propaganda.com');
+    .type('x.com', { delay: 1 });
```

Comment:
1ms per keystroke, overriding the default value of 10ms

Reply:
🤯